platform/binance/transaction.go

```diff
@@ -71,7 +89,7 @@ func normalizeSingleTransfer(srcTx ExplorerTxs, address, token string) blockatla
 		return blockatlas.TxPage{tx}
 	}
 
-	if srcTx.TxAsset == token {
+	if srcTx.TxAsset != "" {
```

Comment:
toke can be anything in this case, not exact match

Reply:
what do you mean?

Reply:
Your changes introduced serious bug:
`curl --location --request GET 'http://localhost:8420/v1/binance/bnb14cjy0yl23xkf0hnw3ql295v8nghqstvlzkvqpl'`
return BEP2 transfers also.